### PR TITLE
Update README.md about the protocol used in the broker address.

### PR DESCRIPTION
**Pull request by kubagrzelak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The following module configurations can be provided in a data service designer s
 
 | Name                | Environment Variables | Type    | Description                |
 | ------------------- | --------------------- | ------- | -------------------------- |
-| MQTT Broker Address | MQTT_BROKER           | string  | eg: test.mosquitto.org     |
+| MQTT Broker Address | MQTT_BROKER           | string  | eg: mqtt://test.mosquitto.org (please notice that the protocol used in the module must be explicitely stated. Supported protocols: `mqtt://` or `ws://`)  |
```
Comment (europaul) [inline README.md:42]: I don't think we support websocket here, nor should we. Could you check it it works with MQTTS and port 8883?

**Review by europaul — CHANGES_REQUESTED** [review]

File: image/src/app.js [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
 const { broker, port, topic, protocol } = require('./config/broker')
 const { urls } = require('./config/nextContainer.js')
-const { module_name, module_type } = require('./config/thisContainer.js')
 const fetch = require('node-fetch')
 
-const client = mqtt.connect(broker, {
-  port,
-  protocol,
-})
+if(broker.includes('tcp') || broker.includes('tls') || broker.includes('ws') || broker.includes('wss') || broker.includes('wxs') || broker.includes('alis')) {
```
Comment (europaul) [inline image/src/app.js:6]: it's easier to whitelist the supported protocols, than try to blacklist all the unsupported ones. also using simple `includes` can heavily backfire: e.g. mqtt://browser.com contains "ws"

File: image/src/app.js [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+  console.error("Unsupported url protocol. Please provide mqtt:// or mqtts:// urls.")
```
Comment (europaul) [inline image/src/app.js:7]: idk javascript, but it would be nice to have this line terminate the app instead of having a huge `else` block

**Review by europaul — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kubagrzelak) [reply README.md]: removed

File: image/src/app.js (hunk 2 above)
Comment (kubagrzelak) [reply image/src/app.js]: That's right. I added a new approach

**Review by europaul — APPROVED** [review]
I haven't tested it, but looks good!

**Pull request merged**